src/GitTabExpansion.ps1

```diff
@@ -29,20 +29,18 @@ $gitflowsubcommands = @{
 }
 
 function script:gitCmdOperations($commands, $command, $filter) {
-    $commands.$command -split ' ' |
-        Where-Object { $_ -like "$filter*" }
+    $commands.$command -split ' ' | Where-Object { $_ -like "$filter*" }
 }
 
-
 $script:someCommands = @('add','am','annotate','archive','bisect','blame','branch','bundle','checkout','cherry',
                          'cherry-pick','citool','clean','clone','commit','config','describe','diff','difftool','fetch',
-                         'format-patch','gc','grep','gui','help','init','instaweb','log','merge','mergetool','mv',
-                         'notes','prune','pull','push','rebase','reflog','remote','rerere','reset','revert','rm',
+                         'format-patch','gc','grep','gui','help','init','instaweb','ls-files','log','merge','mergetool',

```

Comment:
Out of curiosity, why add `ls-files`? You can get tab expansion for all commands (including plumbing like `ls-files`) by setting `$Global:GitTabSettings.AllCommands = $true`, but I'm not sure this belongs with the porcelain commands.